flight/Modules/Actuator/actuator.c

```diff
@@ -444,6 +445,43 @@ static float MixerCurve(const float throttle, const float* curve, uint8_t elemen
 	return curve[idx1] * (1.0f - scale) + curve[idx2] * scale;
 }
 
+/* curve is defined in N intervals connecting N+1 points
```

Comment:
Can this to be factored to be common code and take the range as parameters to avoid duplication of code?